internal/pkg/cli/svc_init.go

```diff
@@ -573,8 +589,8 @@ func (o *initSvcOpts) askDockerfile() error {
 
 func (o *initSvcOpts) askSvcPort() (err error) {
 	// If the port flag was set, use that and don't ask.
-	if o.port != 0 {
-		return nil
+	if len(o.ports) > 0 {
+		return o.validateMultiplePorts()
```

Comment:
So there are two workflows:
1. `copilot init` that does validation before ask phase. 
2. `copilot svc init` does validation first phase then ask for service type as well as port. So we need to check at this point if the multiple port use case is valid? Because port validation at validation phase doesn't check this condition as `wkldType`is unknown at that point. Does that make sense?